js/jquery.keyboard.extension-navigation.js

```diff
@@ -132,6 +132,7 @@ $.fn.addNavigation = function(options){
 		};
 
 		base.getMaxIndex = function(vis, row) {
+			var kbcss = $.keyboard.css;
```

Comment:
Hi @tlu200!

Thanks for fixing this, but I think it might be easier to add that definition after `kbevents` (line 81), then we can remove all the extra definitions of `kbcss` in the file.

If you are willing to make the change that'd be awesome! If not, I'll work on it soon...